app/models/service/resource_linking.rb

```diff
@@ -0,0 +1,36 @@
+module Service::ResourceLinking
+  extend ActiveSupport::Concern
+
+  def add_provider_vms(options)
+    # required keys in options hash:
+    #   :target_id, the id of the provider to refresh
+    #   :array_of_uid_ems, an array of VMs' uid_ems
+    #
+    options = {
+      :target_class => "ExtManagementSystem",
+      :name         => "Linking VMs to service #{name} ID: [#{id}]",
+      :userid       => evm_owner.userid,
+      :sync_key     => guid,
+      :service_id   => id,
+      :zone         => my_zone
+    }.merge(normalize_inputs!(options))
+
+    _log.info("NAME [#{options[:name]}] for user #{evm_owner.userid}")
+
+    job = Service::LinkingWorkflow.create_job(options)
+    return job
```

Comment:
No need for the `return job` line, the result of `Service::LinkingWorkflow.create_job` will be returned from the method unless it is raising an error.

The previous line should just be: `Service::LinkingWorkflow.create_job(options)`